docs/INSTALL.md

```diff
@@ -53,36 +113,6 @@ sudo dnf install -y \
 brew install cmake qt@5
 ```
 
-### Getting sources
-
-Clone the repository:
-
-```bash
-git clone --recursive https://github.com/organicmaps/organicmaps.git
-```
-
-Default clone destination directory is organicmaps.
-
-Update git submodules (sometimes doesn't work automatically):
-
-```bash
-git submodule update --init --recursive
-```
-
-Configure the repository as opensource build:
-
-```bash
-./configure.sh
-```
-
-or with private repository
-
-```bash
-./configure.sh <private-repo-name>
-```
-
-You can check usage `./configure.sh --help`
-
 ### Building
 
 With all tools installed, just run `tools/unix/build_omim.sh`.
```

Comment:
-r for release build, -d for debug one

This command builds for the desktop.

Reply:
These options are present in the doc already, they're just not visible in the diff.